examples/additive_manufacturing/sintering_physics/README.md

```diff
@@ -0,0 +1,153 @@
+<!-- markdownlint-disable MD033 -->
+
+# Virtual Foundary GraphNet
+
+## Setup with Modulus package
+
+- Download Modulus, make or install
+
+- Find the matching torch-scatter version with torch and cuda version enabled:
+  - i.e. pip install torch-scatter-f `https://data.pyg.org/whl/torch-2.2.0%2Bcu121/torch_scatter-2.1.2%2Bpt22cu121-cp311-cp311-linux_x86_64.whl`
+    (replace the torch-scatter wheel with the matching cuda, torch version )
+  - torch-scatter installation guide: `https://pypi.org/project/torch-scatter/`
+  - wheels source: `https://data.pyg.org/whl/`
+
+- pip install tensorflow
+
+  - test version: tensorflow-2.15.0.post1-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+
+- for logging:
+  - pip install wandb
+  - pip3 install mlflow
+
+- For training with mixed precision:
+  - `https://github.com/NVIDIA/apex`
+- pyvista is required only if need to run data proprocessing with the raw
+simulation data files
+  
+- Dev:
+
+  - install pytest
+  - pip install importlib-metadata
+
+## Train
+
+Change the params in constants.py for training:
```

Comment:
Let's use Hydra configs.

Reply:
Let's address this in another PR: https://github.com/NVIDIA/modulus/issues/404

Reply:
Solved